errors/daemon.go

```diff
@@ -372,7 +372,7 @@ var (
 	// volume specification isn't valid.
 	ErrorCodeVolumeInvalid = errcode.Register(errGroup, errcode.ErrorDescriptor{
 		Value:          "VOLUMEINVALID",
-		Message:        "Invalid volume specification: %s",
+		Message:        "Invalid volume specification: '%s'",
```

Comment:
it's more likely that we remove all these error that someone walks our code looking to change `%s`s to `%q`s.